src/core/utils/DefaultModelObjects.ts

```diff
@@ -0,0 +1,50 @@
+import { SubscriptionsCollection } from "../../onesignal/subscriptions/SubscriptionsCollection";
+import { AppConfig } from "../../shared/models/AppConfig";
+import { IdentityModel, UserProperties } from "../types";
+
+/**
```

Comment:
Check what browsers still use ES5, consider dropping support

Reply:
https://caniuse.com/es5

With very low numbers of ES5 browsers, we will drop support